src/app/pages/crypto-asset-list/components/fungible-token-asset-item.tsx

```diff
@@ -0,0 +1,22 @@
+import { BoxProps } from '@stacks/ui';
+
+import type { StacksFungibleTokenAssetBalance } from '@shared/models/crypto-asset-balance.model';
+
+import { StacksFungibleTokenAssetItem } from '@app/components/crypto-assets/stacks/fungible-token-asset/stacks-fungible-token-asset-item';
+
+interface FungibleTokenAssetItemProps extends BoxProps {
+  assetBalance: StacksFungibleTokenAssetBalance;
+}
+export function FungibleTokenAssetItem(props: FungibleTokenAssetItemProps) {
+  const { assetBalance, ...rest } = props;
+  const { blockchain } = assetBalance;
+
+  switch (blockchain) {
+    case 'stacks':
+      return (
+        <StacksFungibleTokenAssetItem assetBalance={assetBalance} isPressable {...(rest as any)} />
```

Comment:
I did a bit on digging on this, seems it's related to emotion?

Also, just a heads up, I believe spreading props as any deactivates typechecking for the rest of the props, [TS Playground Example](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgIilQ3wG4Aoc4AOxiSk3STgAUcwBnOAb3LjhABPNhDAAuOBxhQaAcwoBfSpgCu1DMAjU4AYQAUYdhwkjOASh6V+RGCqjaAPGAB8mCBAcB6F+SXk0WlJw1BAwplwAvDwhMACywuwSAOQARkl+AdQcEAA2SAB0ORCyeg46cCg52QDqONSyEfgpKFD4PPkdejHhFVwo1IJmCnCezmYU5EA).

Although not edited in this PR, I believe this component has a type error which isn't being surfaced due to itself also using `rest as any` internally.

Reply:
It surfaces when using a `forwardRef`. I recall there's a `forwardRefWithAs` in the style lib that fixes this.

Reply:
Yeah, this will be removed soon when we abandon the asset search so I just left for now. Can remove soon.

Reply:
The `fowardRef` is only needed for use with downshift.